go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/fluent/fluent-logger-golang v1.5.0
 	github.com/go-sql-driver/mysql v1.5.0
 	github.com/gofrs/uuid v3.2.0+incompatible
-	github.com/golang/protobuf v1.3.5
+	github.com/golang/protobuf v1.4.0
```

Comment:
This upgrade is what is causing the Spanner tests to fail. Will you revert this (`go get github.com/golang/protobuf@v1.3.5` and `go mod tidy`)?

Reply:
Done